docker-compose/README.md

```
@@ -0,0 +1,266 @@
+To run the latest beta version of Exosphere with the newest features, replace container tags with `beta-latest`:
+
+### Quick Beta Setup with Cloud MongoDB
+```bash
+# Download and modify for beta with cloud MongoDB
+curl -O https://raw.githubusercontent.com/exospherehost/exospherehost/main/docker-compose/docker-compose.yml
+sed -i 's/:latest/:beta-latest/g' docker-compose.yml
+# Set your MONGO_URI in .env file, then:
+docker compose -f docker-compose.yml up -d
+```
+
+### Quick Beta Setup with Local MongoDB
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)